cloudflare/resource_cloudflare_access_policy_test.go

```diff
@@ -697,3 +697,120 @@ func testAccessPolicyOktaConfig(resourceID, zone, accountID string) string {
 
 	`, resourceID, zone, accountID)
 }
+
+func TestAccessPolicyPurposeJustification(t *testing.T) {
```

Comment:
we'll also need to unset API_TOKEN in this test unless this endpoint supports it.